src/pfs/pfs.proto

```diff
@@ -284,6 +294,31 @@ message DeleteBranchRequest {
   bool force = 2;
 }
 
+message CreateProjectRequest {
+  Project project = 1;
+  string description = 2;
+  bool update = 3;
+}
+
+message CreateProjectResponse {}
+
+message InspectProjectRequest {
+  Project project = 1;
+}
+
+message InspectProjectResponse {
+  ProjectInfo project_info = 1;
+}
+
+message ListProjectRequest {}
+
+message DeleteProjectRequest {
+  Project project = 1;
+  bool force = 2;
+}
+
+message DeleteProjectResponse {}
```

Comment:
Similar to above, delete calls don't normally have a response type. There could potentially be value in this, but I would favor consistency with the other rpcs personally.